primitives/io/src/lib.rs

```diff
@@ -89,30 +76,6 @@ pub trait Storage {
 		self.storage(key).map(|s| s.to_vec())
 	}
 
-	/// All Child api uses :
-	/// - A `child_storage_key` to define the anchor point for the child proof
-	/// (commonly the location where the child root is stored in its parent trie).
-	/// - A `child_storage_types` to identify the kind of the child type and how its
-	/// `child definition` parameter is encoded.
-	/// - A `child_definition_parameter` which is the additional information required
-	/// to use the child trie. For instance defaults child tries requires this to
-	/// contain a collision free unique id.
-	///
-	/// This function specifically returns the data for `key` in the child storage or `None`
-	/// if the key can not be found.
-	fn child_get(
-		&self,
-		child_storage_key: &[u8],
-		child_definition: &[u8],
-		child_type: u32,
-		key: &[u8],
-	) -> Option<Vec<u8>> {
-		let storage_key = child_storage_key_or_panic(child_storage_key);
-		let child_info = ChildInfo::resolve_child_info(child_type, child_definition)
-			.expect("Invalid child definition");
-		self.child_storage(storage_key, child_info, key).map(|s| s.to_vec())
-	}
-
```

Comment:
Kusama has the crowdfund module using child trie, but it has not been linked to the runtime so it is ok.
But the Edgeware case is more preoccupying to me, I will restore the old IO as deprecated for now.

Reply:
While restoring the old api, I realize it was a broken design: using a child type we cannot add new child type later as it will break the past behavior, so even if I do not like all those change, this is good one (runtime level, I am really bugged by the rpc still).

Reply:
Actually previous version was correct, we cannot break pas behavior as there was no child trie of the new type existing, so api with child type is right as is api where storage key type is defined by prefix. I will probably switch everything back to prefixed storage key.